Src/Scripts/UnitDefinitions/Viscosity.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+    "Name": "Viscosity",
+    "BaseUnit": "Centistokes",
```

Comment:
If we go with Stokes (which I have no idea if is a good base unit or not), then we have the benefit of reusing the prefix templating such as "centi", "kilo", "mega" etc.
From skimming through the wiki, it seems there are different types of viscosity such as kinematic, dynamic etc. Should we distinguish between these? Such as DynamicViscosity, KinematicViscosity? Or can you convert between kinematic viscosity units and dynamic viscosity units? I honestly don't know.